### PR TITLE
Improve appearance names

### LINKED ISSUE #276: GlossyCarPaint and MatteCarPaint are too specific
GlossyCarPaint and MatteCarPaint appearances are slightly used in other contexts.
They are basically a flat surface painted with a glossy or matte paint. Or metallized paint?

I propose to simply remove the "Car" keyword from their names.
@all Is it ok like this, or do you have better propositions?

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fix #276

- [x] Rename `MatteCarPaint` -> `MattePaint`
- [x] Rename `GlossyCarPaint` -> `GlossyPaint`
- [x] Rename `CarLeather` -> `Leather`
- [x] Update doc accordingly
- [x] Update icons accordingly
- [x] Add missing `Leather` to the icon studio
- [x] Log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
👍

File: projects/appearances/protos/GlossyPaint.proto [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # license url: http://www.apache.org/licenses/LICENSE-2.0
 # A glossy car paint material. The color can be selected in the `baseColor` field. Useful with any of the vehicle PROTOs.
```
Comment (DavidMansolino) [inline projects/appearances/protos/GlossyPaint.proto]:
```suggestion
# A glossy paint material. The color can be selected in the `baseColor` field.
```

File: projects/appearances/protos/Leather.proto [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # tags: static
 # A car leather material. The color can be overridden using the `colorOverride` field.
```
Comment (DavidMansolino) [inline projects/appearances/protos/Leather.proto]:
```suggestion
# A leather material. The color can be overridden using the `colorOverride` field.
```

File: projects/appearances/protos/MattePaint.proto [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # license url: http://www.apache.org/licenses/LICENSE-2.0
 # A matte car paint material. The color can be selected in the `baseColor` field. Useful with any of the vehicle PROTOs.
```
Comment (DavidMansolino) [inline projects/appearances/protos/MattePaint.proto]:
```suggestion
# A matte paint material. The color can be selected in the `baseColor` field.
```

File: docs/guide/appearances.md [hunk 4]
```diff
@@ -73,45 +73,6 @@ BrushedAluminium {
 
 - `IBLStrength`: Defines the strength of ambient lighting from the Cubemap node.
 
-## CarLeather
```
Comment (DavidMansolino) [inline docs/guide/appearances.md]: This should not be edited manually.

Comment (fabienrohrer) [reply docs/guide/appearances.md]: Thank you, I just applied the script and it seems to work smoothly.

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**